DQMServices/Core/src/MonitorElement.cc

```diff
@@ -200,21 +201,29 @@ MonitorElement::MonitorElement(const MonitorElement &x, MonitorElementNoCloneTag
 MonitorElement::MonitorElement(const MonitorElement &x)
   : MonitorElement::MonitorElement(x, MonitorElementNoCloneTag())
 {
+  std::lock(mutex_, x.mutex_);
```

Comment:
There is no need to lock the `this->mutex_` since you are in a constructor so by design no code can have access to this object yet.